### ISSUE TITLE
Variables provisioning_model and instance_termination_action in spot_instance_basic module 

### ISSUE THREAD

**Issue by kimuracl** [terraform-google-modules/terraform-docs-samples#192]
### TL;DR

Variables provisioning_model and instance_termination_action in spot_instance_basic module not working with provider hashicorp/google v3.90.1.

### Expected behavior

When I use "terraform validate" command, I expect how to answer "Success! The configuration is valid." 

Example:

$ terraform validate
Success! The configuration is valid.

### Observed behavior

When I use "terraform validate" command, i receive the next answer:

 $ terraform validate
╷
│ Error: Unsupported argument
│
│   on main.tf line 15, in resource "google_compute_instance" "spot_vm_instance":
│   15:       provisioning_model = "SPOT"
│
│ An argument named "provisioning_model" is not expected here.
╵
╷
│ Error: Unsupported argument
│
│   on main.tf line 16, in resource "google_compute_instance" "spot_vm_instance":
│   16:       instance_termination_action = "STOP"
│
│ An argument named "instance_termination_action" is not expected here.
╵

### Terraform Configuration

```hcl
# main.tf
resource "google_compute_instance" "spot_vm_instance" {
  name         = "spot-instance-name"
  machine_type = "f1-micro"
  zone         = "us-central1-a"

  boot_disk {
    initialize_params {
      image = "debian-cloud/debian-11"
    }
  }

  scheduling {
      preemptible = true
      automatic_restart = false
      #provisioning_model = "SPOT"
      #instance_termination_action = "STOP"
  }

  network_interface {
    # A default network is created for all GCP projects
    network = "default"
    access_config {
    }
  }
}

# provider.tf
terraform {
  required_version = ">= 0.13.1"
  required_providers {
    google = {
      source  = "hashicorp/google"
      version = "~>3.85"
    }
  }
}
```

### Terraform Version

```sh
$ terraform version
Terraform v1.3.1
on darwin_amd64
+ provider registry.terraform.io/hashicorp/google v3.90.1
```

### Additional information

No more comments.
